tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationView.java

```diff
@@ -256,10 +263,28 @@ private TableViewerColumn createTableViewerColumn(
         column.setWidth( bound );
         column.setResizable( true );
         column.setMoveable( true );
+        column.addSelectionListener(getSelectionAdapter(column, colNumber));
 
         return viewerColumn;
     }
 
+    private SelectionAdapter getSelectionAdapter( final TableColumn column, final int index )
```

Comment:
This block has a mixture of spaces and tabs, turn on "show whitespace characters" and you can clearly see the mixuture of spaces and tabs, we want all spaces.